File: Data Structures & Algorithms/string-encode-and-decode/submission-2.py
class Solution:

    def encode(self, strs: List[str]) -> str:
        en = ""

        for s in strs:
            en = en + str(len(s)) + '#' + s

        return en


    def decode(self, s: str) -> List[str]:
        res = []
        i=0

        while i < len(s):
            j = i
            while s[j] != '#':
                j += 1

            length = int(s[i:j])
            i = j+1
            j = i + length
            res.append(s[i:j])
            i = j

        return res
        

File: Data Structures & Algorithms/two-integer-sum-ii/submission-2.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int n = numbers.size();
        vector<int> res;
        int i =0, j = 0;
        while(i<n ){
            j = i+1;
            while(j<n){

                if(numbers[i] + numbers[j] == target){
                    return {i+1,j+1};
                }
                j++;
            }
            i++;
        }
        return {0,0};
    }
};
